src/bp/core/routers/telemetry.ts

```diff
@@ -0,0 +1,50 @@
+import * as sdk from 'botpress/sdk'
+import { TelemetryRepository } from 'core/repositories/telemetry'
+import AuthService, { TOKEN_AUDIENCE } from 'core/services/auth/auth-service'
+import { RequestHandler, Router } from 'express'
+
+import { CustomRouter } from './customRouter'
+import { checkTokenHeader } from './util'
+
+export class TelemetryRouter extends CustomRouter {
+  private checkTokenHeader: RequestHandler
+
+  constructor(
+    private logger: sdk.Logger,
+    private authService: AuthService,
+    private telemetryRepo: TelemetryRepository
+  ) {
+    super('Telemetry', logger, Router({ mergeParams: true }))
+    this.checkTokenHeader = checkTokenHeader(this.authService, TOKEN_AUDIENCE)
+    this.setupRoutes()
+  }
+
+  private setupRoutes() {
+    // TODO get server header
```

Comment:
Hmm ?

Reply:
Like I said in https://github.com/botpress/botpress/pull/3662#discussion_r463592565. It is for a future pr.